### DISCUSSION TITLE
pulseaudio 15.0

### DISCUSSION THREAD

**Discussion by mcnaugha** [Homebrew/discussions#2689] [Everyday usage]
pulseaudio 15.0 adds support for LDAC and AptX HD. The bottled version is currently 14.2. How do we get it updated to 15.0 or is there a way we can build 15.0 under homebrew for macOS?

15.0 is a significant release. With the integrated support for LDAC and AptX HD there is a possibility to support Bluetooth Audio receivers (e.g. LDAC-enabled headphones) that support these codecs and output higher bit rate audio from Apple Music to such receivers. This would be a major first for the Mac.

Cannot believe how long Apple is taking to deliver its own solution to the problem. Qualcomm just announced AptX Lossless codec. Apple has to have a trick up its sleeve. I'd like to imagine that it's to say goodbye Bluetooth Audio and hello Wi-Fi audio or something so that there is no need for new codecs or lossy formats.

**Comment by gromgit** [comment]
> How do we get it updated to 15.0

Someone has to figure out how to solve the build failures in [this 5-month-old PR](https://github.com/Homebrew/homebrew-core/pull/82113).
